testssl.sh

```diff
@@ -2417,7 +2430,7 @@ neat_list(){
           fi
           return 0
      fi
-     read what_dh bits <<< "$kx"
+     read what_dh bits <<< "$(echo "$kx" | awk -F"(" '{a=$1" ("$2; print a}')"
```

Comment:
Not sure about the reason of the change here but if there's one, one minor thing: I would rather avoid a pipe like  ``read what_dh bits <<< "$(awk -F"(" '{a=$1" ("$2; print a}') <<< "$kx" "